portfolio/src/main/webapp/script.js

```diff
@@ -52,6 +52,23 @@ function seeLess(id) {
     const moreId = id + 'More';
     const lessId = id + 'Less';
 
-    document.getElementById(lessId).style.display = "block";
-    document.getElementById(moreId).style.display = "none";
+    document.getElementById(lessId).style.display = 'block';
+    document.getElementById(moreId).style.display = 'none';
 }
+
+function getComments() {
+    fetch('/data').then(response => response.json()).then((messages) => {
+        console.log(messages);
+        const commentsList = document.getElementById('comments-container');
+        // commentsList.innerHTML = '';
```

Comment:
Prefer deleting lines over commenting, unless there is a specific reason you want to comment out.